src/Components/Scenes/Governance/DomainManagerTable.tsx

```diff
@@ -8,23 +8,26 @@ import {
   addDomainOwnersRequestFetch
 } from '../../../State/Governance/actions';
 import { TextField } from '@material-ui/core';
+import SnackBar from '../../Presentational/Modal/SnackBar';
```

Comment:
So will we use this SnackBar for all our notifications? I have one created as well in the Hydration App.

Reply:
Yeah Maybe we should combine them into one.